webxr-api/device.rs

```diff
@@ -40,4 +41,6 @@ pub trait Device: 'static {
     /// While this method is being called, the device has unique access
     /// to the texture. The texture should be sync'd using glWaitSync before being used.
     fn render_animation_frame(&mut self, texture_id: u32, size: Size2D<i32>, sync: GLsync);
+
+    fn initial_inputs(&self) -> Vec<InputInfo>;
```

Comment:
Bikeshedding: not sure about the name. `input_sources: Vec<InputSource>`? Needs some documentation too.

Reply:
This is about the initial input sources only, we don't handle changes yet. The client will have to keep track of changes.

Reply:
Still needs documentation!